include/emitc/emitc_tosa.h

```diff
@@ -49,6 +49,63 @@ inline Src add(Src x, Src y) {
   return emitc::add<Src>(x, y);
 }
 
+/// Functions for TOSA binary elementwise elementwise ops.
+// MulOp
+template <typename Src>
+inline Src mul(Src x, Src y, int32_t shift) {
+  return emitc::mul(x, y);
+}
+
+/// Functions for other TOSA ops.
+// BroadcastInDimOp
+template <typename Dest, typename Src>
+inline Dest
+broadcast_in_dim(Src operand,
+                 Tensor<int64_t, Src::rank()> broadcast_dimensions) {
+  return emitc::broadcast_in_dim<Dest>(operand, broadcast_dimensions);
+}
```

Comment:
There is no `broadcast_in_dim` in TOSA. Thus, we shouldn't define one in our header. Just directly convert to an emitc call that is calling the core func.